projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -469,4 +485,30 @@ export class IgxComboComponent extends IgxComboBaseDirective implements AfterVie
             selection.join(', ');
         return value;
     }
+
+    private applySelection(oldSelection: any[]): void {
```

Comment:
This method is not really needed since we already have the `setSelection` method. It does more or less the same thing and the only difference is that in the `setSelection` we also emit an event to notify that the selection has been applied and this can be toggled with an additional argument that says whether it should emit or not.